app/lib/frontend/templates.dart

```diff
@@ -447,38 +447,43 @@ class TemplateService {
       }).toList();
     }
 
+    final executables = selectedVersion.pubspec.executables?.keys?.toList();
+    executables?.sort();
+    final hasExecutables = executables != null && executables.isNotEmpty;
+    final onlyGlobalInstall = hasExecutables && importExamples.isEmpty;
+
     final exampleVersionConstraint = '"^${selectedVersion.version}"';
 
-    final bool renderGeneric = !isFlutterPackage ||
+    final bool usePubGet = !isFlutterPackage ||
         platforms == null ||
         platforms.isEmpty ||
         platforms.length > 1 ||
         platforms.first != KnownPlatforms.flutter;
 
-    final bool renderFlutter = isFlutterPackage ||
+    final bool useFlutterPackagesGet = isFlutterPackage ||
         (platforms != null && platforms.contains(KnownPlatforms.flutter));
 
-    String toolHtml;
-    if (renderGeneric && renderFlutter) {
-      toolHtml = '<code>pub get</code> or <code>packages get</code>';
-    } else if (renderFlutter) {
-      toolHtml = '<code>packages get</code>';
+    String editorSupportedToolHtml;
+    if (usePubGet && useFlutterPackagesGet) {
+      editorSupportedToolHtml =
+          '<code>pub get</code> or <code>packages get</code>';
```

Comment:
Should the last one be `<code>flutter packages get</code>` ?

Reply:
yeah, it should, fixing...